courses/free-courses-en.md

```diff
@@ -822,6 +822,7 @@
 * [SwiftUI Tutorials for Beginners](https://www.youtube.com/playlist?list=PLMRqhzcHGw1Z-lZaaun3A3mV9PbEfHANI) - CodeWithChris
 * [The Complete Swift iOS Developer - Create Real Apps in Swift](https://www.udemy.com/course/the-complete-ios-10-developer-course/) - Grant Klimaytys (Udemy)
 * [Unit Testing in iOS](https://www.youtube.com/playlist?list=PLMRqhzcHGw1ZLLvLwuW8AP3n-A3nsRn9P) - CodeWithChris
+* [iOS Development Course - Use Swift 5 and UIKit to Build a Netflix Clone](https://www.youtube.com/watch?v=KCgYDCKqato&t=3862s) - Amr, FreeCodeCamp
```

Comment:
We prefer 'playlist' links for YouTube contributions, along with the producer/creator's name. More info in docs/CONTRIBUTING.md#platforms-and-access-notes.


Avoid single video links and remove any timestamps (e.g., &t=3862s) when sharing. 

Follow alphabetical ordering and review the free-programming-books-lint output for linter errors, focusing on alphabetical order and spacing.